string description
uint32 product_code
uint32 pcb
uint32 pca
uint32 serial
uint32 firmware_major
uint32 firmware_minor
